enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/replication/shipping/RaftLogShippingManager.java

```diff
@@ -79,9 +78,23 @@ public RaftLogShippingManager( Outbound<CoreMember,RaftMessages.RaftMessage> out
         membership.registerListener( this );
     }
 
-    public synchronized void start( LeaderContext initialLeaderContext )
+    /**
+     * Paused when stepping down from leader role.
+     */
+    public synchronized void pause()
     {
-        if( destroyed )
+        running = false;
```

Comment:
It is checked in onMembershipChanged(), which is invoked by the membership manager, and protects against starting log shippers when not running.